src/getHumanAge.test.js

```diff
@@ -7,4 +7,49 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array', () => {
+    expect(getHumanAge(0, 0))
+      .toBeInstanceOf(Array);
+  });
+
+  test('should return [0, 0] for inputs (0, 0)', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 0 for inputs < 15', () => {
```

Comment:
```suggestion
  test('should return [0, 0] for inputs < 15', () => {
```

fix it in all of your tests

Reply:
thank you for the review, I have changed most of the tests except for 2 last ones.